### ISSUE TITLE
Bootstrap navbar?

### ISSUE THREAD

**Issue by aslakhellesoy** [cucumber-attic/website#21]
Why doesn't the site use Bootstrap's [navbar](http://getbootstrap.com/components/#navbar)? It's nice and responsive. It makes it easy to [fix it to the top](http://getbootstrap.com/components/#navbar-fixed-top), which I think would be nice since many of the pages under `/docs` are rather long.

I'm sure we can style it to look like the current custom navbar.

**Comment by jbpros** [comment]
The reason is historical. Initially, not all apps were using bootstrap. We started with a common navbar between all apps and sites (cukes.info, cucumber.pro, legacy app, modern, etc.) and couldn't use bootstrap at that time. Now that all pages are based on bootstrap 3, we can switch to its navbar.

TL;DR: +1.

**Comment by romaingweb** [comment]
The switch has been made! 146d2f3
